File: Go/01.Go-Basic-February-2022/06.conditionalStatementAdvanced-Exercise/01.cinema.go
package main

import "fmt"

func main() {
	var typeProjection string
	fmt.Scanln(&typeProjection)

	var rows, cols int
	fmt.Scanln(&rows)
	fmt.Scanln(&cols)

	var countTickets int = rows * cols
	var pricePerTicket float32 = 0

	switch typeProjection {
		case "Premiere":
			pricePerTicket = 12
		case "Normal":
			pricePerTicket = 7.5
		case "Discount":
			pricePerTicket = 5.00
	}

	var profit float32 = float32(countTickets) * pricePerTicket
	fmt.Printf("%.2f leva", profit)
}

File: JavaScript/01.JavaScript-Basic-February-2021/06.conditionalStatementsAdvanced-Exercise/01.cinema.js
function cinema(input) {
    let type = input[0];
    let r = Number(input[1]);
    let c = Number(input[2]);

    let income = 0;
    if (type == "Premiere") {
        income=r*c*12;
    } else if (type == "Normal") {
        income=r*c*7.5;
    } else if (type == "Discount") {
        income=r*c*5;
    }

    console.log(`${income.toFixed(2)} leva`);    
}

cinema(["Premiere", "10", "12"]);
cinema(["Normal", "21", "13"]);
cinema(["Discount", "12", "30"]);